### PR TITLE
Add all China TMA

### PR DESCRIPTION
**Pull request by RudiZhang**

### CODE REVIEW

**Comment by kmoberg** [comment]
Wow, this is impressive work! @RudiZhang Can you email karl@nyartcc.org with a confirmation from VATPRC that you have the authority to apply these changes and I'll merge it ASAP.

Thanks for your contribution!

File: TRACONBoundaries.geojson [hunk 1]
```diff
@@ -492,17 +492,53 @@
...
...[[102.025,35.778055555556],[101.83361111111,35.776944444444],[101.82305555556,35.779027777778],[101.74133333333,35.795555555556],[101.65497222222,35.819111111111],[101.57141666667,35.848611111111],[101.49127777778,35.883833333333],[101.41516666667,35.924555555556],[101.34363888889,35.970444444444],[101.27725,36.021194444444],[101.21647222222,36.076361111111],[101.16183333333,36.135611111111],[101.11366666667,36.198444444444],[101.07244444444,36.264416666667],[101.03841666667,36.333027777778],[101.01188888889,36.40375],[100.99308333333,36.476055555556],[100.98216666667,36.549416666667],[100.97925,36.623277777778],[100.98436111111,36.697055555556],[100.9975,36.770166666667],[101.01855555556,36.842111111111],[101.04744444444,36.912277777778],[101.08388888889,36.980194444444],[101.12772222222,37.04525],[101.17855555556,37.107027777778],[101.23602777778,37.164972222222],[101.29969444444,37.218694444444],[101.36911111111,37.26775],[101.44372222222,37.31175],[101.52294444444,37.350361111111],[101.57577777778,37.374138888889],[101.58333333333,37.375277777778],[102.04111111111,37.438333333333],[103.01638888889,37.520833333333],[102.87,36.520555555556],[102.76527777778,35.782222222222]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:514]: I'm not familiar with this airspace, but there is a significant overlap of airspace here. Can you confirm that this is supposed to be this way? @RudiZhang 
Screen Shot 2022-09-19 at 12 30 21

File: TRACONBoundaries.geojson [hunk 2]
```diff
@@ -492,17 +492,53 @@
...
+    {"type":"Feature","properties":{"id":"ZHEC","prefix":["ZHEC"],"name":"Ezhou Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[115.18666666667,30.971666666667],[115.5325,30.821111111111],[115.5575,30.271666666667],[115.10361111111,29.925277777778],[114.66611111111,29.991944444444],[114.74472222222,30.386111111111],[114.83111111111,30.815555555556]]]]}},
+    {"type":"Feature","properties":{"id":"ZHHH","prefix":["ZHHH"],"name":"Wuhan Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[114.56666666667,29.033333333333],[113.12305555556,29.392222222222],[112.66444444444,29.503055555556],[112.27166666667,30.288333333333],[112.39888888889,31.095833333333],[113.61277777778,32.3275],[114.07,32.125],[114.99138888889,32.539722222222],[115.73305555556,32.467777777778],[115.675,32.183333333333],[115.93333333333,30.083333333333]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:509]: Again - same thing here, there is an overlap between Wuhan Approach and Ezhou Approach. I assume there are different altitudes here, but can you confirm this is correct?

Screen Shot 2022-09-19 at 12 30 36

File: TRACONBoundaries.geojson [hunk 3]
```diff
@@ -492,17 +492,53 @@
...
+    {"type":"Feature","properties":{"id":"ZSJN","prefix":["ZSJN"],"name":"Jinan Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[115.4,36.666666666667],[117.5,37.486666666667],[118.21638888889,37.711944444444],[118.21666666667,36.000555555556],[115.42361111111,36.000555555556]]]]}},
+    {"type":"Feature","properties":{"id":"ZSNB","prefix":["ZSNB"],"name":"Ningbo Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[121.21972222222,30.255],[121.11472222222,30.338333333333],[120.96027777778,30.088611111111],[120.89444444444,29.981666666667],[120.84583333333,29.9025],[120.61305555556,29.693333333333],[120.59194444444,29.425],[122.66666666667,29.338611111111],[122.66666666667,30.241666666667]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:524]: This corner here looks a bit strange - there are multiple borders, and overlap of three or more sectors. Is this correct or possible to clean up?

Screen Shot 2022-09-19 at 12 31 44

**Comment by RudiZhang** [comment]
> Wow, this is impressive work! @RudiZhang Can you email [karl@nyartcc.org](mailto:karl@nyartcc.org) with a confirmation from VATPRC that you have the authority to apply these changes and I'll merge it ASAP.
> 
> Thanks for your contribution!

Done, and I'm the staff of VATPRC also XDXD

File: TRACONBoundaries.geojson [hunk 4]
```diff
@@ -492,17 +492,53 @@
...
+    {"type":"Feature","properties":{"id":"ZGHA","prefix":["ZGHA"],"name":"Changsha Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[113.06833333333,28.816666666667],[114.06527777778,28.700833333333],[113.84833333333,27.631666666667],[113.52861111111,27.628611111111],[112.52333333333,27.733333333333],[112.20944444444,28.066666666667]]]]}},
+    {"type":"Feature","properties":{"id":"ZGJD","prefix":["ZGJD"],"name":"Zhuhai Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[113.48333333333,22.881666666667],[113.55666666667,23.021666666667],[114.11,23.036666666667],[114.22833333333,22.908333333333],[114.75,22.921666666667],[114.75,22.408333333333],[114.5,22.408333333333],[114.23,22.563333333333],[113.82833333333,22.41],[113.665,22.228333333333],[113.69833333333,22.16],[112.72638888889,21],[112.48333333333,22.163333333333],[112.48277777778,22.406944444444],[112.74805555556,22.46],[112.89416666667,22.489166666667],[113.19472222222,22.55],[113.32833333333,22.625]]]]}},
```
Comment (kmoberg) [inline TRACONBoundaries.geojson:502]: The borders are not correct here. They don't match up and have overlap in multiple areas. Could this be fixed, please?

Screen Shot 2022-09-19 at 12 33 41

**Review by kmoberg — CHANGES_REQUESTED** [review]
Incredible work adding all these sectors! Please refer to my comment in the PR, and to the inline comments for some minor questions and change requests.

File: TRACONBoundaries.geojson (hunk 1 above)
Comment (RudiZhang) [reply TRACONBoundaries.geojson:514]: Yep, depends on the altitude.

File: TRACONBoundaries.geojson (hunk 2 above)
Comment (RudiZhang) [reply TRACONBoundaries.geojson:509]: Same, Ezhou Approach is low APP sector, Wuhan Approach will over if Ezhou offline

File: TRACONBoundaries.geojson (hunk 3 above)
Comment (RudiZhang) [reply TRACONBoundaries.geojson:524]: I also confirm it. Same depends on altitude

File: TRACONBoundaries.geojson (hunk 4 above)
Comment (RudiZhang) [reply TRACONBoundaries.geojson:502]: Oh, that is the only thing I missed. Yeah. I will fix that

Comment (RudiZhang) [reply TRACONBoundaries.geojson]: Fix. And overlay area is normal

**Comment by RudiZhang** [comment]
Fixed. @kmoberg

**Pull request merged**